plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -760,7 +762,9 @@ fu_cros_ec_usb_device_write_firmware (FuDevice *device,
 	FuCrosEcFirmware *cros_ec_firmware = FU_CROS_EC_FIRMWARE (firmware);
 	gint num_txed_sections = 0;
 
-	if (fu_device_has_custom_flag (device, "rebooting-to-ro")) {
+	fu_device_remove_private_flag (device, FU_CROS_EC_USB_DEVICE_FLAG_SPECIAL);
```

Comment:
I'm trying to figure out how the "special" custom flag worked before, and if this is the right place to remove it.

Reply:
Interesting. Many of these flags previously got swept up and removed implicitly by the way the custom flags overwrote each other. Now that private flags are a bit more sticky, we can be more explicit with this one and the rebooting to ro.

I'll find the right place to manipulate each of these transitory flags.

Thanks for doing this, @hughsie! This makes this whole thing a lot easier to follow, at the end of all of this.

Reply:
Awesome!  Go ahead and rebase the branch and force push after you're done.

Reply:
Nah, @bleungatchromium just commit a "fixup" commit to the branch and I'll rebase and squash. I know how to resolve the conflict that will happen. :)